queries/config/.graphqlconfig

```diff
@@ -4,7 +4,7 @@
   "extensions": {
     "endpoints": {
       "Default GraphQL Endpoint": {
-        "url": "https://squid.subsquid.io/rubick/v/007/graphql",
+        "url": "https://squid.subsquid.io/rubick/v/008/graphql",
```

Comment:
How about using `URLS.koda.rubick` variable to replace this string?

Reply:
Well this is a json file, I couldn't find any documentation about moving this config file to ts (maybe https://the-guild.dev/graphql/config/docs ?)

Reply:
Aright. Let's keep this so.